### PR TITLE
Package adoption

### PR DESCRIPTION
**Pull request by aynik**
Just to adopt this package under @adgorithmics.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                  context: npm-publish
+                  filters:
+                      tags:
+                          only: /^v[0-9]+(\.[0-9]+)*(-rc[0-9]*)*/
+                      branches:
+                          ignore: /.*/
+            - publish-hot:
+                  requires:
+                      - test
+                  filters:
```
Comment (cgsimmons) [inline .circleci/config.yml]: need context added here as well

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            - image: circleci/node:10.13.0
+
+        steps:
+            - *pin-npm-version
+            - checkout
+
+            - restore_cache:
+                  keys:
+                      - jscache-{{ checksum "package.json" }}
+                      - jscache-
```
Comment (lukas-reineke) [inline .circleci/config.yml]: is this correct?

Comment (cgsimmons) [reply .circleci/config.yml]: yes, it is a fallback if the checksum fails

File: package.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "react-tagsinput",
+  "name": "@adgorithmics/react-tagsinput",
   "version": "3.19.0",
   "description": "Highly customizable React component for inputing tags",
   "main": "react-tagsinput.js",
   "peerDependencies": {
-    "react": "^16.0.0 || ^15.0.0 || ^0.14.0"
+    "react": "16.6.3"
```
Comment (3upzorz) [inline package.json]: can you make this `>= 16`? we're gonna get peer dep warning with an exact version here

File: .circleci/config.yml (hunk 2 above)
Comment (aynik) [reply .circleci/config.yml]: It is used as a fallback mechanism. See https://circleci.com/docs/2.0/caching/#restoring-cache

File: package.json (hunk 3 above)
Comment (aynik) [reply package.json]: I'm pairing it with https://github.com/adgorithmics-inc/adgo-ui/blob/master/package.json#L79

Comment (3upzorz) [reply package.json]: @aynik I moved this isnt a peer dependency in another open PR in adgo-ui, it's not correct to have it as a direct dependency

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    publish:
+        docker:
+            - image: circleci/node:10.13.0
+
+        steps:
+            - *pin-npm-version
+            - checkout
+            - add_ssh_keys:
+                  fingerprints:
+                      - 'd6:6b:f3:95:10:ae:27:b1:fb:09:d9:f6:55:c3:b0:dc'
```
Comment (cgsimmons) [inline .circleci/config.yml]: This needs to be changed to the fingerprint for the deploy key in this repo
`8e:ff:a1:31:90:fc:fc:8e:bb:fb:e0:ac:a7:f6:49:d2`

Comment (aynik) [reply .circleci/config.yml]: @cgsimmons fixed

**Comment by cgsimmons** [comment]
LGTM

**Pull request merged**